vere/sist.c

```diff
@@ -660,7 +805,10 @@ _sist_rest()
       u3_lo_bail();
     }
 
-    if (u3r_mug('h') != led_u.mag_l ) {
+    if ( u3r_mug('g') == led_u.mag_l ) {
```

Comment:
That was the original code, but it doesn't work because `led_u` isn't re-read out of the log after it's upgraded. The relationship between `_sist_rest()` and `_sist_rest_nuu()` would need to be more broadly refactored for this kind of validation to work. But I think it's fine for `_sist_rest()` to just trust `_sist_reset_nuu()`.